### PR TITLE
feat(rewards-component): redesign

### PR DESCRIPTION
**Pull request by worrex**
## Related Issues

---

## Checklist:

- [ ] I have read [Migaloo's contribution guidelines](https://github.com/White-Whale-Defi-Platform/migaloo-frontend/blob/main/CONTRIBUTING.md).
- [ ] My pull request has a sound title and description (not something vague like `Update index.md`)
- [ ] All existing and new tests are passing.
- [ ] I updated/added relevant documentation.
- [ ] The code is formatted properly `yarn lint`.
- [ ] The project builds and is able to deploy on Netlify `yarn build`.

### CODE REVIEW

File: components/Pages/Dashboard/BondingActions/hooks/useTransaction.tsx [hunk 1]
```diff
@@ -144,7 +144,7 @@ export const useTransaction = () => {
         message = 'Out of gas, try increasing gas limit on wallet.'
       } else if ((/before the new\/latest epoch/u).test(e?.toString())) {
         setError('Epoch not yet created.')
-        message = 'Please force the pending epoch on the dashboard.'
+        message = 'Please force the pending epoch on the bonding.'
```
Comment (Sen-Com) [inline components/Pages/Dashboard/BondingActions/hooks/useTransaction.tsx:147]: "on the bonding page" or something

**Review by nick134-bit — APPROVED** [review]
rename dashboard on mobile at a later time

**Comment by Sen-Com** [comment]
Bildschirmfoto 2024-01-30 um 17 34 42

The bullet points look shabby at best

File: components/Pages/Dashboard/Dashboard.tsx [hunk 2]
```diff
@@ -219,6 +219,20 @@ const Dashboard: FC = () => {
             />
           </VStack>
         </Flex>
+        <VStack alignItems={'start'} width={'full'} pt={11}>
```
Comment (Sen-Com) [inline components/Pages/Dashboard/Dashboard.tsx:222]: ????

**Review by Sen-Com — CHANGES_REQUESTED** [review]

**Review by SebastianJer7 — APPROVED** [review]

**Pull request merged**